xrootd/xrootd-stageout-server/xrootd-redir-docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+ervices:
+  xrootd-stageout-server:
+    image: cmscaltech/xrootd-stageout-server:cmsdev
+    network_mode: host
+    restart: unless-stopped
+    environment:
+    # Redirector configuration
+    #- XRD_REDIR=redir-01.sense.unl.edu
+    - XRD_REDIR=red-sense-dtn2.unl.edu
+    #- XRD_REDIR=198.32.43.1
+    - XRD_REDIR_PORT=1213
+    # Local XRootD Ports
+    - XRD_PORT=1095
+    - XRD_MNG_PORT=1213
+    # Local FS point. This has to match with volume mounted
+    - LOCAL_ROOT=/storage/cms/
+    # Sitename, mainly for reporting
+    - SITENAME=T2_US_Nebraska_Test
+    # Certificate/Key
+    - XRD_CERT=/etc/grid-security/xrootdcert.pem
+    - XRD_KEY=/etc/grid-security/xrootdkey.pem
+    # XRootD http secret key. Mandatory if HTTPREADWRITE is set
+    - XRD_HTTP_SECRET_KEY=pYrySWhj4BftwJkSbMyAk8ha3p5YXsAt7g3mFzX7Vkg
```

Comment:
This is security issue. You should remove all files and change http secret key.

Reply:
@juztas Changed the file and deleted the key